app/modules/transactions/components/transaction_next_action.svelte

```diff
@@ -0,0 +1,152 @@
+<script>
+  import Spinner from '#components/spinner.svelte'
+  import Flash from '#lib/components/flash.svelte'
+  import { getNextActionsData } from '#transactions/lib/next_actions'
+  import { updateTransactionState } from '#transactions/lib/transactions_actions'
+  import { I18n, i18n } from '#user/lib/i18n'
+
+  export let transaction
+
+  $: nextActions = getNextActionsData(transaction)
```

Comment:
I guess its the same bug, but just in case, documenting same behaviour about `Confirm`
![2024-02-22_08-06](https://github.com/inventaire/inventaire-client/assets/5363918/a52ec335-4318-45d3-834c-c0824848ecbb)

Reply:
The `transaction.state` attribute was not being updated, that should be fixed by 99fb8a237, which depends on https://github.com/inventaire/inventaire/pull/733